postgresql/resource_postgresql_extension.go

```diff
@@ -47,6 +49,12 @@ func resourcePostgreSQLExtension() *schema.Resource {
 				Computed:    true,
 				Description: "Sets the version number of the extension",
 			},
+			extDatabaseAttr: {
```

Comment:
You need to set `ForceNew` to true. If we change the database we need to remove the extension from the old one and create it in the new one. (it's impossible to update)